File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/AttributeSetInstanceExtensionFieldMvoStateEventId.java
package org.dddml.wms.domain;

import org.dddml.wms.domain.AttributeSetInstanceExtensionFieldId;

public class AttributeSetInstanceExtensionFieldMvoStateEventId
{
    private AttributeSetInstanceExtensionFieldId attributeSetInstanceExtensionFieldId;

    public AttributeSetInstanceExtensionFieldId getAttributeSetInstanceExtensionFieldId()
    {
        return this.attributeSetInstanceExtensionFieldId;
    }

    public void setAttributeSetInstanceExtensionFieldId(AttributeSetInstanceExtensionFieldId attributeSetInstanceExtensionFieldId)
    {
        this.attributeSetInstanceExtensionFieldId = attributeSetInstanceExtensionFieldId;
    }

    private Long attrSetInstEFGroupVersion;

    public Long getAttrSetInstEFGroupVersion()
    {
        return this.attrSetInstEFGroupVersion;
    }

    public void setAttrSetInstEFGroupVersion(Long attrSetInstEFGroupVersion)
    {
        this.attrSetInstEFGroupVersion = attrSetInstEFGroupVersion;
    }

    public String getAttributeSetInstanceExtensionFieldIdGroupId()
    {
        return getAttributeSetInstanceExtensionFieldId().getGroupId();
    }

    public void setAttributeSetInstanceExtensionFieldIdGroupId(String attributeSetInstanceExtensionFieldIdGroupId)
    {
        getAttributeSetInstanceExtensionFieldId().setGroupId(attributeSetInstanceExtensionFieldIdGroupId);
    }

    public String getAttributeSetInstanceExtensionFieldIdIndex()
    {
        return getAttributeSetInstanceExtensionFieldId().getIndex();
    }

    public void setAttributeSetInstanceExtensionFieldIdIndex(String attributeSetInstanceExtensionFieldIdIndex)
    {
        getAttributeSetInstanceExtensionFieldId().setIndex(attributeSetInstanceExtensionFieldIdIndex);
    }


}

File: Dddml.Wms.AdminUI/ts/AttributeSetInstanceExtensionFieldMvoStateEventId.ts
namespace Dddml.Wms.Domain {

	export class AttributeSetInstanceExtensionFieldMvoStateEventId {

		private attributeSetInstanceExtensionFieldId: AttributeSetInstanceExtensionFieldId;

		getAttributeSetInstanceExtensionFieldId(): AttributeSetInstanceExtensionFieldId {
			if(!this.attributeSetInstanceExtensionFieldId) {
				this.attributeSetInstanceExtensionFieldId = new AttributeSetInstanceExtensionFieldId(); 
			}
			return this.attributeSetInstanceExtensionFieldId;
		}

		setAttributeSetInstanceExtensionFieldId(attributeSetInstanceExtensionFieldId: AttributeSetInstanceExtensionFieldId) {
			this.attributeSetInstanceExtensionFieldId = attributeSetInstanceExtensionFieldId;
		}

		private attrSetInstEFGroupVersion: string;

		getAttrSetInstEFGroupVersion(): string {
			return this.attrSetInstEFGroupVersion;
		}

		setAttrSetInstEFGroupVersion(attrSetInstEFGroupVersion: string) {
			this.attrSetInstEFGroupVersion = attrSetInstEFGroupVersion;
		}

	}
}

